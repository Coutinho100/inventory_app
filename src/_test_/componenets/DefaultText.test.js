import React from 'react';
import DefaultText from '../../components/Component/DefaultText';
import { View, Text } from 'react-native';
import { configure, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import test_function from './test_function';
configure({ adapter: new Adapter() });

test_function(["<text render/>",1],['renders correctly','renders the length of view'],[<DefaultText />, Text])
