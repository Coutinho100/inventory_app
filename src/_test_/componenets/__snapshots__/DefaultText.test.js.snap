// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<text render/> renders correctly 1`] = `
<Text
  style={
    Object {
      "fontFamily": "open-sans",
      "textAlign": "justify",
    }
  }
>
   
</Text>
`;
